manifests/s/sylikc/JPEGView/1.1.41/sylikc.JPEGView.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: sylikc.JPEGView
+PackageVersion: 1.1.41
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0

```

```suggestion
MinimumOSVersion: 10.0.0.0
AppsAndFeaturesEntries:
  - DisplayVersion: 1.1.41.0
```
For exact matching with Control Panel / Registry.